scripts/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "${RUNNER_NAME}" ]; then
+  echo "RUNNER_NAME must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_TOKEN}" ]; then
+  echo "RUNNER_TOKEN must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_ORG}" ]; then
+  echo "RUNNER_ORG must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_REPO}" ]; then
+  echo "RUNNER_REPO must be set" 1>&2
+  exit 1
+fi
+
+cd /runner
+mkdir -p _work
+./config.sh --unattended --replace --name "${RUNNER_NAME}" --url "https://github.com/${RUNNER_ORG}/${RUNNER_REPO}" --token "${RUNNER_TOKEN}" --work /runner/_work
+
+# TODO: run placemat
+
+./bin/runsvc.sh
```

Comment:
Why do we use `bin/runsvc.sh`? Github instructs to execute `run.sh` that is in the same directory as `config.sh`.

Reply:
`bin/runsvc.sh` keeps running when the runner restarts due to [update](https://github.com/actions/runner/blob/8109c962f09d9acc473d92c595ff43afceddb347/src/Misc/layoutbin/RunnerService.js#L64), while `bin/run.sh` does not..
This script is invoked after the time-consuming initializing step, so it's better to use `bin/runsvc.sh` and keep the process running.